uswds-jekyll.gemspec

```diff
@@ -1,19 +1,19 @@
 # coding: utf-8
 
-Gem::Specification.new do |spec|
-  spec.name          = "uswds-jekyll"
-  spec.version       = "2.1.0"
-  spec.authors       = ["Shawn Allen"]
-  spec.email         = ["shawn.allen@gsa.gov"]
+Gem::Specification.new do |s|
+  s.name          = 'uswds-jekyll'
+  s.version       = '2.2.0'
+  s.authors       = ['Shawn Allen, Brian Hurst, Scott Weber']
```

Comment:
I think the expected format for the `authors` and `email` is `s.authors = ['John Jones', 'Mary Smith']`